frontend/src/main/components/Nav/AppNavbar.js

```diff
@@ -13,7 +13,7 @@ export default function AppNavbar({ currentUser, systemInfo, doLogout, currentUr
           <AppNavbarLocalhost url={currentUrl} />
         )
       }
-      <Navbar expand="xl" variant="dark" bg="dark" sticky="top" data-testid="AppNavbar">
+      <Navbar expand="xl" variant="dark" className="color-nav" sticky="top" data-testid="AppNavbar">
```

Comment:
This seems unrelated.  We try to keep a PR to a single topic.

I suggest you build two separate branches by using git cherry-pick to separate these into a pr for the fixtures, and a separate PR for the color change.  Each of those is worth 5 pts separately.

If you intended to put them into one PR, then your PR title and description need to spell that out; but many organizations frown on that (i.e. having two or more unrelated concerns in a single PR.)

Reply:
I have a separate PR for the navBar changes currently, I would just like to remove those two commits from this PR without breaking it. I tried to undo the two commits about the NavBar but it ended up deleting .js files needed to run the site. If there is an easy way to roll those out of this PR I would like to know how.

Reply:
Nevermind, I resolved this in a new PR.